app/src/main/java/com/example/messengerAndroid/utils/Constants.kt

```diff
@@ -0,0 +1,18 @@
+package com.example.messengerAndroid.utils
+
+object Constants {
+
+    // constants for Validator
+    const val PASSWORD_PATTERN = "^(?=.*[a-z])(?=.*[A-Z])(?=.*\\d)[a-zA-Z\\d]{8,}$"
+    const val EMAIL_PATTERN = "([\\w.\\-_]+?@\\w+?\\x2E.+)"
```

Comment:
I can't comment in the Validator.kt class, so I'll write it here. Try this: in getValidityEmail() in Validator.kt
`return Patterns.EMAIL_ADDRESS.matcher(email).matches()`

Reply:
thank you, did not know about this class, actually it is pattern for password, either, but in sdk 26 and above